.changeset/lemon-carrots-leave.md

```diff
@@ -0,0 +1,9 @@
+---
+"angular-workspace": patch
+"@astrouxds/angular": patch
+"astro-website": patch
+"@astrouxds/react": patch
+"@astrouxds/astro-web-components": patch
+---
+
+Submitting spacing work on clock (so far)
```

Comment:
We can delete this changeset. The audience for changesets is the end user via our [release notes](https://github.com/RocketCommunicationsInc/astro/releases/tag/v6.12.1)